local-startup/running-proxy-node-locally.md

```diff
@@ -0,0 +1,81 @@
+# Running proxy node locally
+
+This directory contains resources to run a Proxy Node deployment locally with Docker. This is a first attempt and while a whole journey can be successfully completed, there are certain limitations:
+
+- The Translator (and all other components) is configured to use a file credential configuration, which means it uses a set of pk8 files on the disk to sign and encrypt messages. Hence, the HSM functionality is not tested. The next step would be to investigate whether we can run a [SoftHSM](https://www.opendnssec.org/softhsm/) instance locally to simulate a real HSM for the Translator.
```

Comment:
We should discuss the SoftHSM stuff, as it wasn't appropriate when we tried to use it before as its implementation is vastly different from the AWS HSM implementation.

Reply:
Yeah we might not be able to use it. I've not looked into it in much detail but we should definitely look into it.